nexus/src/app/sagas/disk_create.rs

```diff
@@ -970,6 +1020,7 @@ pub(crate) mod test {
         let sled_agent = &cptestctx.sled_agent.sled_agent;
         let datastore = cptestctx.server.apictx().nexus.datastore();
 
+        assert!(no_stuck_sagas(&cptestctx.logctx.log, datastore).await);
```

Comment:
I'm assuming yes, but was this failing before the non-test changes?

Reply:
Not if you apply this patch on `main`, but I did see stuck saga(s) that prompted this check.